modules/lvms-uninstalling-logical-volume-manager-operator-using-rhacm.adoc

```diff
@@ -6,198 +6,29 @@
 [id="lvms-uninstalling-lvms-rhacm_{context}"]
 = Uninstalling {lvms} installed using {rh-rhacm}
```

Comment:
@radeore it'd be great if you could verify this as well during the installation verification.

Reply:
RHACM procs are being tracked in this PR - https://github.com/openshift/openshift-docs/pull/73279